packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/AppContext.kt

```diff
@@ -320,6 +320,13 @@ class AppContext(
     return UIManagerHelper.getUIManagerForReactTag(reactContext, viewTag)?.resolveView(viewTag) as? T
   }
 
+  /**
+   * Runs a code block on the javascript thread
+   */
+  fun runOnJS(runnable: Runnable) {
```

Comment:
First of all, I think it's important to add "thread" to the function name, for clarity.
What would you say about `executeOnJavaScriptThread`/`executeOnJSThread`?
We already have `runOnQueue` on the AsyncFunction component, which has much different purpose so I think it kinda makes sense to distinguish them 🤔
On the other side, we can also consider changing `runOnQueue` to something else like `setQueue` or `attachToQueue` 🤔